v2/src/styles/settings.scss

```diff
@@ -76,24 +76,25 @@ $shadow-key-umbra-opacity: rgba(24, 43, 75, 0.1);
 $shadow-key-penumbra-opacity: 0%;
 $shadow-key-ambient: 0%;
 
-$shadow-key-umbra: 
+$shadow-key-umbra:
 (
-    0: (0px 0px 0px 0px $shadow-key-umbra-opacity),    
+    0: (0px 0px 0px 0px $shadow-key-umbra-opacity),
     1: (0px 8px 12px 0px $shadow-key-umbra-opacity)
 );
- 
+
 /**
 --- Border radius
 */
 $rounded:
 (
     'xl': 12px,
+    'sm': 4px
```

Comment:
vuetify comes with  rounded by 4px by default.  i don't think you need to edit that one. But i am pretty indifferent to it. so we can keep it